### ISSUE TITLE
use websockets to speed up ESI UI

### ISSUE THREAD

**Issue by tzumainn** [CCI-MOC/esi#626]

**Comment by tzumainn** [comment]
@ajamias can you populate the description of this issue? thanks!

**Comment by ajamias** [comment]
@tzumainn I created a draft PR that sends node data to the nodes panel using websockets here: [https://github.com/CCI-MOC/esi-ui/pull/26](https://github.com/CCI-MOC/esi-ui/pull/26) for proof of concept. If this is satisfactory, then I'll clean it up and make it work for offers as well.

**Comment by joachimweyl** [comment]
@tzumainn What are the next steps for this?

**Comment by tzumainn** [comment]
I had left some comments on the PR; I believe Austin is working on an update.

**Comment by ajamias** [comment]
Yes, I am still working on this. Sorry progress has been slow recently..

**Comment by tzumainn** [comment]
I'll move it to the product backlog.
